packages/openassistants/openassistants/functions/base.py

```diff
@@ -120,13 +121,18 @@ def get_entities(self) -> Sequence[IEntity]:
         return self.entities
 
 
+class BaseFunctionParameters(BaseModel):
+    json_schema: JSONSchema = EMPTY_JSON_SCHEMA
+
+
 class BaseFunction(IBaseFunction, BaseModel, abc.ABC):
     id: str
     type: str
     display_name: Optional[str] = None
     description: str
     sample_questions: List[str] = []
     confirm: bool = False
+    parameters: BaseFunctionParameters = BaseFunctionParameters()
```

Comment:
Can we make this optional?

Reply:
it'll be set to a EMPTY_JSON_SCHEMA

i don't want it to be none to get a consistent behavior

Reply:
Agree it's effectively optional like this